Source/WebCore/platform/network/ResourceHandleInternal.h

```diff
@@ -108,20 +98,11 @@ class ResourceHandleInternal {
     bool m_defersLoading;
     bool m_shouldContentSniff;
     ContentEncodingSniffingPolicy m_contentEncodingSniffingPolicy;
-#if USE(CFURLCONNECTION)
-    RetainPtr<CFURLConnectionRef> m_connection;
-    ResourceRequest m_currentRequest;
-    RefPtr<ResourceHandleCFURLConnectionDelegate> m_connectionDelegate;
-#endif
 #if PLATFORM(COCOA)
     RetainPtr<NSURLConnection> m_connection;
     RetainPtr<id> m_delegate;
-#endif
-#if PLATFORM(COCOA)
-    bool m_startWhenScheduled { false };
-#endif
-#if PLATFORM(COCOA) || USE(CFURLCONNECTION)
     RetainPtr<CFURLStorageSessionRef> m_storageSession;
+    bool m_startWhenScheduled { false };
```

Comment:
nit: putting this with other bools would pack better.

Reply:
Oh! I will do!